src/proxy/show.go

```diff
@@ -79,8 +79,8 @@ func (spanner *Spanner) handleShowTableStatus(session *driver.Session, query str
 	router := spanner.router
 
 	database := session.Schema()
-	if !ast.Database.IsEmpty() {
-		database = ast.Database.Name.String()
+	if ast.Database != "" {
```

Comment:
this change is for?

Reply:
> this change is for?

Avoid `show table status from tt.t`, this statement should report a syntax error.